koku/masu/processor/ocp/ocp_cost_model_cost_updater.py

```diff
@@ -417,6 +418,17 @@ def _delete_tag_usage_costs(self, start_date, end_date, source_uuid):
                 filters={"monthly_cost_type": "Tag", "report_period_id": report_period_id},
             )
 
+    def distribute_costs_and_update_ui_summary(self, start_date, end_date):
+        """Distribute cost model costs and update UI summary tables"""
+        with OCPReportDBAccessor(self._schema) as accessor:
+            accessor.populate_distributed_cost_sql(start_date, end_date, self._provider_uuid, self._distribution_info)
+            accessor.populate_ui_summary_tables(start_date, end_date, self._provider.uuid)
+            report_period = accessor.report_periods_for_provider_uuid(self._provider_uuid, start_date)
+            if report_period:
+                with schema_context(self._schema):
+                    report_period.derived_cost_datetime = timezone.now()
+                    report_period.save()
```

Comment:
the dbaccessor sets the schema, so this can be removed:
```suggestion
                report_period.derived_cost_datetime = timezone.now()
                report_period.save()
```